core/src/main/java/bio/terra/pearl/core/service/participant/EnrolleeService.java

```diff
@@ -279,6 +284,11 @@ public String generateShortcode() {
         return shortcode;
     }
 
+    @Transactional
+    public List<Enrollee> findGovernedEnrollees(List<EnrolleeRelation> enrolleeRelations) {
```

Comment:
since this doesn't care what the relation type is
```suggestion
    public List<Enrollee> findRelatedEnrollees(List<EnrolleeRelation> enrolleeRelations) {
```